packages/components/psammead-script-switch/src/index.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import styled from 'styled-components';
+import { getPica } from '@bbc/gel-foundations/typography';
+import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+import { C_WHITE, C_EBON } from '@bbc/psammead-styles/colours';
+import { string, shape, node } from 'prop-types';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+
+const SIZE_ABOVE_600PX = '3rem'; // 48px
+const SIZE_BELOW_600PX = '2.5rem'; // 40px
+
+const getDimensions = size => `
+  height: ${size}
+  width: ${size}
+`;
+
+const StyledLink = styled.a`
+  ${({ script }) => script && getPica(script)}
+  ${({ service }) => service && getSansRegular(service)}
+
+  display: inline-block;
+  border: 0.0625rem solid ${C_WHITE};
+  color: ${C_WHITE};
+  text-align: center;
+  text-decoration: none;
+
```

Comment:
Can you add `box-sizing: border-box;` to avoid the `div` to expand when hover over it?

Reply:
![ezgif com-video-to-gif](https://user-images.githubusercontent.com/48688870/66125813-9f5b8e00-e5f0-11e9-89cd-73d5f09f7faa.gif)
Interesting. I don't see it expanding at all on my end on hover.

Reply:
I think because you are using `width`, once you replace it with the paddings, you should notice it.